backend/src/generated/tmp-resolvers/index.ts

```diff
@@ -0,0 +1,16 @@
+// This resolver file was scaffolded by github.com/prisma/graphqlgen, DO NOT EDIT.
+// Please do not import this file directly but copy & paste to your application code.
+
+import { Resolvers } from "../graphqlgen";
+
+import { Query } from "./Query";
+import { Post } from "./Post";
+import { User } from "./User";
+import { Mutation } from "./Mutation";
+
+export const resolvers: Resolvers = {
+  Query,
+  Post,
+  User,
+  Mutation
+};
```

Comment:
We create one file "Resolver" with all the resolvers in it. It's not wrong (we guess) but makes a bit harder to read and find what you are looking for

Reply:
For the current use case, a single resolver is totally fine.
But in bigger project it totally makes sense to have multiple. 
Seems like `no-js-pls` wanted to exercise that a bit :)